### PR TITLE
add dockerfile

### PR DESCRIPTION
**Pull request by coder2z**

### CODE REVIEW

**Comment by coder2z** [comment]
I have read the CLA Document and I hereby sign the CLA

File: Makefile [hunk 1]
```diff
@@ -32,6 +33,7 @@ submodules:
 	# -rm -rf install/submodules/* && git submodule update --remote
 
 image:
-	@mkdir -p bin
-	@GOOS=linux go build -o bin/ trpc/trpc.go
-	@docker build -t trpc-cmdline:${VERSION} .
+	@docker build \
+	--build-arg GOPROXY="https://goproxy.cn" \
```
Comment (WineChord) [inline Makefile:37]: Should we set this as the default? Alternatively, should we offer it as a separate build target or as an optional feature?

Comment (coder2z) [reply Makefile]: modify GOPROXY,GOSUMDB to be optional

**Review by WineChord — APPROVED** [review]
LGTM

**Pull request merged**